packages/nextjs/src/config/webpack.ts

```diff
@@ -78,10 +79,10 @@ export function constructWebpackConfigFunction(
     // will call the callback which will call `f` which will call `x.y`... and on and on. Theoretically this could also
     // be fixed by using `bind`, but this is way simpler.)
     const origEntryProperty = newConfig.entry;
-    newConfig.entry = async () => addSentryToEntryProperty(origEntryProperty, options.isServer);
+    newConfig.entry = async () => addSentryToEntryProperty(origEntryProperty, buildContext);
```

Comment:
Why pass in the entire build context now?

Reply:
Oh - I guess we don't need to anymore. In an earlier draft of the real PR (the one this is prep for), I was also checking webpack version, and handling webpack 4 differently than webpack 5. To your point above about webpack changing things version to version, I'm inclined to leave it, in case sometime in the future we need to bring that version check back in.

Reply:
Makes sense to me 👍